resources/views/store/cart/_item.blade.php

```diff
@@ -13,7 +13,7 @@
     <input type="hidden" name="item[id]" value="{{ $item->id }}">
 
     <span class="store-order-item__name">
-        {{ $item->getDisplayName() }}
+        {!! $item->getDisplayName() !!}
```

Comment:
call `blade_safe(...)` for supporter tag display name instead

Reply:
Done